.github/workflows/rpc-compat.yml

```diff
@@ -0,0 +1,34 @@
+name: rpc-compat
```

Comment:
can you find a way to do this as a test instead of a separate job? we're right at capacity in terms of parallel jobs and adding another one will impact our ability to scale CI

Reply:
Could the jobs here be added to the existing `openapi` workflow? We need to generate schema for both `master` and the current branch to check for breaking changes.

Reply:
yeah that probably makes the most sense since it's basically the same goals but different

Reply:
Sounds good.

Reply:
this was marked resolved but is still an issue so I'm marking it unresolved and leaving a comment